test/presentation/feature/news/top_anime_page_test.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_template_riverpod/data/api/api_client.dart';
+import 'package:flutter_template_riverpod/data/model/api/response/get_top_anime_response.dart';
+import 'package:flutter_template_riverpod/data/providers/my_anime_list_repository_provider.dart';
+import 'package:flutter_template_riverpod/data/repository/my_anime_list_repository.dart';
+import 'package:flutter_template_riverpod/presentation/feature/news/top_anime_page.dart';
+import 'package:flutter_template_riverpod/presentation/feature/news/widgets/top_anime_item_widget.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+
+import 'top_anime_page_test.mocks.dart';
+
+Finder _gridItemText(int id) =>
```

Comment:
better name can be
_findGridItemTitleText

Reply:
Thank you, I corrected it.